### PR TITLE
update apiMetadata.js with V4_1 info architecure

### PR DESCRIPTION
**Pull request by VivianAllen**
update metadata object to reflect new info architecture and API database contents

### CODE REVIEW

File: src/data/apiMetadata.js [hunk 1]
```diff
@@ -436,263 +841,160 @@ export default [
...
+          code: "QS104EW0001",
+          name: "",
+          slug: "",
+        },
       },
       {
+        code: "KS103EW",
+        name: "Marital status",
+        slug: "marital-status",
+        desc: "People married of in civil partnerships.",
```
Comment (PeteMontgomery) [inline src/data/apiMetadata.js:871]: Should be `or` not `of`

File: src/data/apiMetadata.js [hunk 2]
```diff
@@ -436,263 +841,160 @@ export default [
...
+            code: "KS103EW0004",
+            name: "In a registered civil partnership",
+            slug: "in-a-registered-civil-partnership",
+          },
           {
             code: "KS103EW0005",
-            name: "Separated",
-            slug: "separated",
+            name: "Separated (but still legally married)",
+            slug: "separated-but-still-legally-married",
```
Comment (PeteMontgomery) [inline src/data/apiMetadata.js:891]: How can you be illegally married?

**Review by PeteMontgomery — DISMISSED** [review]

File: src/data/apiMetadata.js (hunk 1 above)
Comment (VivianAllen) [reply src/data/apiMetadata.js]: fixed

File: src/data/apiMetadata.js (hunk 2 above)
Comment (VivianAllen) [reply src/data/apiMetadata.js:905]: Apparently so - forced marriages and so on

**Review by PeteMontgomery — APPROVED** [review]

**Pull request merged**